docs/next/guides/building-and-testing/plugins.md

```diff
@@ -157,9 +157,6 @@ export class CustomPlugin extends BasePlugin {
   }
 
   /**
-   * The `updatePlugin` method is called on the `afterUpdateSettings` hook
```

Comment:
This is still technically true, but only if the config object passed to `updateSettings` contains entries that make the plugin relevant to the update. Maybe we could add a short info on that here?

Reply:
@jansiegel - right, how about this? [e5e1af4](https://github.com/handsontable/handsontable/pull/9271/commits/e5e1af4a8d13596f7e1f49cf6637c8fbf99d10b5)

Reply:
👌